packages/mobile/src/components/atoms/ErrorFallback/index.tsx

```diff
@@ -0,0 +1,26 @@
+import $ from "./style.module.scss";
+
+type Props = {
+  error: Error;
+  reset: () => void;
+  height: string;
+};
+
+function ErrorFallback(props: Props) {
+  const { error, reset, height } = props;
+  return (
+    <div className={$["error-fallback"]} style={{ ...{ height } }}>
```

Comment:
여기 스타일 속성이 `height` 밖에 없다면 비구조화를 쓰지 않아도 될 것 같아요!

```suggestion
    <div className={$["error-fallback"]} style={{ height }}>
```